pytripgui/view/chart_widget.py

```diff
@@ -0,0 +1,44 @@
+from PyQt5.QtChart import QChart, QChartView, QLineSeries
+from PyQt5.QtGui import QPainter
+from PyQt5.QtCore import Qt
+
+
+class ChartWidget:
+    def __init__(self):
+        self.model = QChart()
+        self.view = QChartView()
+
+        self.view.setChart(self.model)
+
+        self.model.legend().setAlignment(Qt.AlignBottom)
+        self.view.setRenderHint(QPainter.Antialiasing)
+
+    def show(self):
+        self.view.show()
+
+    def exit(self):
+        self.view.close()
+
+    @property
+    def title(self):
+        return self.model.title()
+
+    @title.setter
+    def title(self, title):
+        self.model.setTitle(title)
+
+    def add_series(self, x, y, name):
+        series = QLineSeries()
+        series.setName(name)
+
+        for x, y in zip(x, y):
```

Comment:
It is bad idea to use same variable names for iteration as for arguments in `zip` function.
Better would be:
```
for xlist, ylist in zip(x,y):
    series.append(xlist, ylist)
```